www/caddy/src/opnsense/www/js/widgets/Metadata/Caddy.xml

```diff
@@ -14,7 +14,8 @@
     <caddycertificate>
         <filename>CaddyCertificate.js</filename>
         <endpoints>
-            <endpoint>/api/caddy/*</endpoint>
+            <endpoint>/api/caddy/diagnostics/certificate</endpoint>
+            <endpoint>/api/caddy/reverse_proxy/get</endpoint>
```

Comment:
Here * may make more sense

Reply:
Makes more sense indeed. The linter is very specific though, it could suggest * instead of the specific endpoint.

Reply:
make lint-acl
Registered widget endpoints do not match:
<<<<<<< src/opnsense/www/js/widgets/CaddyCertificate.js
/api/caddy/diagnostics/certificate
/api/caddy/reverse_proxy/get
========
/api/caddy/diagnostics/*
/api/caddy/reverse_proxy/*
>>>>>>> src/opnsense/www/js/widgets/Metadata/Caddy.xml
*** Error code 1

Reply:
Sounds good. Only wildcards are commited here right now, which makes lint fail. If the specific endpoints are better, I will revert the last commit here.

Reply:
Yes, checked and ACL.xml is fine and house only use one specific endpoint for each controller so change it back to what appeases the lint-acl. Sorry for the noise. Phone review is hard. 😔

Reply:
Well don't stress yourself so much this could have had time later too. xD